wordpress-mobile/WordPress-Android
# Dashboard Personalization: Activity Card track More Menu events

Parent #18948 

This PR adds tracking events for the following;
- "More Menu" tapped
- "More Menu - All activity" tapped
- "More Menu - Hide this" tapped

Not included in this PR is hooking in the logic to update the preference on "hide this" tap.

### **To test:**
**Setup**
- Install the app
- Login and chose a site that has access to Activity Log
- Navigate to Me tab > App Settings > Privacy Settings > Enable "collect information"

**Test Events Tracked**
- Navigate to My Site tab
- Scroll down until you reach the "Recent Activity" card
- Tap on the "More Menu"
- ✅ Verify the log contains 🔵 Tracked: my_site_dashboard_contextual_menu_accessed, Properties: {"card":"activity_log"}
- Tap on the "All Activity" menu item
- ✅ Verify the log contains 🔵 Tracked: my_site_dashboard_card_menu_item_tapped, Properties: {"card":"activity_log","item":"all_activity"}
- Navigate back to the dashboard
- Tap on the "More Menu" > "Hide This"
- ✅ Verify the log contains 🔵 Tracked: my_site_dashboard_contextual_menu_accessed, Properties: {"card":"activity_log"}
- ✅ Verify the log contains🔵 Tracked: my_site_dashboard_card_menu_item_tapped, Properties: {"card":"activity_log","item":"hide_this"}

## Regression Notes
1. Potential unintended areas of impact
The events aren't tracked correctly

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit tests

3. What automated tests I added (or what prevented me from doing so)
New tests were added to `CardsTrackerTest`
PR submission checklist:


UI Changes testing checklist: N/A
